contracts/whitelist/src/contract.rs

```diff
@@ -196,6 +247,18 @@ mod tests {
         setup_contract(deps.as_mut());
     }
 
+    #[test]
+    fn check_start_time_after_end_time() {
```

Comment:
Maybe test for start time is expired as well?

Reply:
https://giphy.com/gifs/titanic-ship-it-Hw8vYF4DNRCKY